docusaurus.config.js

```diff
@@ -85,10 +86,10 @@ const config = {
   ],
   stylesheets: [
     {
-      href: "https://cdn.jsdelivr.net/npm/katex@0.16.2/dist/katex.min.css",
-      type: "text/css",
-      integrity: "sha256-oWCabCfPd4Oi21wqZezBSz/anto4VYcJqc9sM9IzQTk=",
-      crossorigin: "anonymous",
+      href: 'https://cdn.jsdelivr.net/npm/katex@0.13.24/dist/katex.min.css',
```

Comment:
why is the version going down? :confused:

Reply:
The docusaurus docs have a note about version compatibility: https://docusaurus.io/docs/markdown-features/math-equations#configuration

It says something like "use the exact same versions of these packages". It could mean only in the package installation steps, but I just copy/pasted all the relevant bits from that page.

Reply:
ah gotcha. good context, thanks!